src/explore-education-statistics-frontend/src/components/ButtonLink.tsx

```diff
@@ -19,13 +19,11 @@ const ButtonLink = ({
   variant,
   ...props
 }: Props) => {
-  const isAbsolute = typeof to === 'string' && to.startsWith('http');
-
-  const link = (
-    <a
-      // eslint-disable-next-line react/jsx-props-no-spreading
+  return (
+    <Link
+      href={to}
+      prefetch={prefetch}
```

Comment:
Explicit props should go after the spread to avoid being accidentally overridden in the future i.e.

```tsx
<Link
  {...props}
  href={to}
  prefetch={prefetch}
  classNames={...}
/>
```

Reply:
Good spot, fixed